generators/client/templates/angular/src/main/webapp/app/shared/tracker/_tracker.service.ts

```diff
@@ -11,8 +12,8 @@ import { AuthServerProvider } from '../auth/auth-oauth2.service';
 <%_ } _%>
 
 <%_ // TODO find a better way to import these libs here _%>
```

Comment:
remove this comment if the below import works. I tried it once but it didnt work. Are you sure it works now?